lib/oli/authoring/editing/activity_editor.ex

```diff
@@ -717,6 +717,7 @@ defmodule Oli.Authoring.Editing.ActivityEditor do
       |> Enum.reduce(%{}, fn t, m -> Map.put(m, t.id, t) end)
 
     AuthoringResolver.from_resource_id(project_slug, activity_ids)
+    |> Enum.filter(fn r -> !is_nil(r) end)
```

Comment:
This would be easier to read as: `Enum.filter(&(not is_nil(&1)))` (may have to tweak syntax)